actions/allPictures.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
Maybe it's better to leave one empty string after "use strict" to increase readability?